### PR TITLE
reliability to use table in database instead of xml file. fixed queri…

### PR DESCRIPTION
**Pull request by ridz1208**
…es and restructured class

### CODE REVIEW

**Comment by ridz1208** [comment]
Requires documentation and modification of the project config.xml files

**Comment by christinerogers** [comment]
@ridz1208  nice work!  There should be a php script provided enabling existing projects to migrate their settings from the config.xml file, adn the docs/config.xml (and test/config.xml file) should have removed Reliability tagsets.  Also due to migration step/script - tagging this Caveat for Existing Projects

**Comment by ridz1208** [comment]
Noted, thank you Christine. Some work is still needed before this is ready.

File: SQL/0000-00-00-schema.sql [hunk 1]
```diff
@@ -2165,3 +2165,16 @@ CREATE TABLE `data_release_permissions` (
...
+
+DROP TABLE IF EXISTS `reliability_instruments`;
+CREATE TABLE `reliability_instruments` (
+  `Rel_instID` int(11) NOT NULL AUTO_INCREMENT,
+  `TestID` int(11) NOT NULL,
+  `Visit_label` varchar(255) DEFAULT NULL,
+  `Target_scope` enum('Cross','Within') NOT NULL,
+  `Target_siteID` int(11) DEFAULT NULL,
+  `Threshold` float(4,2) NOT NULL,
+  `Reliability_portion` float (2,2) NOT NULL,
```
Comment (xlecours) [inline SQL/0000-00-00-schema.sql]: Should be called Reliability_ratio to be consistent with the PHP variable name.

File: tools/populate_reliability_instruments.php [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                    $this->insertIfMissing($TestID,$Threshold);
+                }
+
+
+            };
+        }
+    }
+
+    /**
+     * function getting the test name associated with a test ID
```
Comment (ridz1208) [inline tools/populate_reliability_instruments.php]: fix documentation section

File: tools/populate_reliability_instruments.php [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+require_once __DIR__ . "/../vendor/autoload.php";
+require_once 'generic_includes.php';
+require_once 'Database.class.inc';
+require_once 'Utility.class.inc';
+
+/**
+ * Class to implement logic which populates table.
```
Comment (ridz1208) [inline tools/populate_reliability_instruments.php]: change comment

File: modules/reliability/php/NDB_Menu_Filter_reliability.class.inc [hunk 4]
```diff
@@ -227,289 +333,796 @@ class NDB_Menu_Filter_reliability extends NDB_Menu_Filter
...
+                JOIN test_names t ON t.ID = ri.TestID",
+            array()
+        );
+        foreach($thresholds as $row){
+            $list_of_thresholds[$row['Test_name']] = $row['Threshold'];
+        }
         return $list_of_thresholds;
     }
+
     function _setDataTableRows($count)
```
Comment (ridz1208) [inline modules/reliability/php/NDB_Menu_Filter_reliability.class.inc]: This is where the main issue was leading to this revamp

**Comment by ridz1208** [comment]
This was assigned to @stellarxo to help develop the testing database as well as testing the module using the new DB

**Comment by stellarxo** [comment]
@ridz1208 You can find the database on redmine > Loris > Documents > LDK
Or we can look at it together when I get back

**Comment by ridz1208** [comment]
will not be ready for release.

**Pull request closed without merging**